Gems/AWSCore/Code/Tests/Configuration/AWSCoreConfigurationTest.cpp

```diff
@@ -53,75 +51,26 @@ class AWSCoreConfigurationTest
 
         m_awsCoreConfiguration = AZStd::make_unique<AWSCore::AWSCoreConfiguration>();
 
-        m_normalizedSourceProjectFolder = AZStd::string::format("%s/%s%s/", AZ::Test::GetCurrentExecutablePath().c_str(),
-            "AWSResourceMappingManager", AZ::Uuid::CreateRandom().ToString<AZStd::string>(false, false).c_str());
-        AzFramework::StringFunc::Path::Normalize(m_normalizedSourceProjectFolder);
-        m_normalizedSetRegFolderPath = AZStd::string::format("%s/%s/",
-            m_normalizedSourceProjectFolder.c_str(), AZ::SettingsRegistryInterface::RegistryFolder);
-        AzFramework::StringFunc::Path::Normalize(m_normalizedSetRegFolderPath);
-
-        m_localFileIO->SetAlias("@projectroot@", m_normalizedSourceProjectFolder.c_str());
-
-        CreateTestSetRegFile(TEST_VALID_RESOURCE_MAPPING_SETREG);
+        CreateFile(m_setRegFilePath.Native(), TEST_VALID_RESOURCE_MAPPING_SETREG);
+        m_localFileIO->SetAlias("@projectroot@", GetTestTempDirectoryPath().Native().c_str());
     }
 
     void TearDown() override
     {
-        RemoveTestFile();
-        RemoveTestDirectory();
+        RemoveFile(m_setRegFilePath.Native());
```

Comment:
Curious: Is the file created at the root? If not, is there any directory you need to clean up during TearDown() as well?

Reply:
file created in test temp folder which is located in system temp folder and automatically gets cleaned up. Refer to AZ::Test::ScopedAutoTempDirectory

Reply:
Sounds good.